sdcm/cluster.py

```diff
@@ -2567,7 +2567,9 @@ def run_nodetool(self, sub_cmd, args="", options="", timeout=None,
         :return: Remoter result object
         """
         cmd = self._gen_nodetool_cmd(sub_cmd, args, options)
-
+        volume_type = self.parent_cluster.params.get("data_volume_disk_type")
+        if volume_type == 'gp3':
+            timeout = timeout * 5 if timeout else 300
```

Comment:
Now all nodetool commands would have 5 times timeout, even decommission.

I'm not sure it's the effect we want

the issue was with flush, we should focus on that specific command

Reply:
ok @fruch , fixed to be applied only for `flush` command.
Yet i recall @fgelcer raised a concern about more potential nodetool commands that may fail for default timeout as well.

Reply:
> ok @fruch , fixed to be applied only for `flush` command. Yet i recall @fgelcer raised a concern about more potential nodetool commands that may fail for default timeout as well.

yes, but that will be solved when, in the test flow, you call the `run_nodetool()` with the timeout, based on whatever you prefer... this "change" not necessarily needs to be inside `run_nodetool()`